crates/ra_assists/src/add_explicit_type.rs

```diff
@@ -1,10 +1,7 @@
-use hir::{
-    HirDisplay, Ty,
-    db::HirDatabase,
-};
+use hir::{HirDisplay, Ty, db::HirDatabase};
```

Comment:
Hm, we explicitelly disable import reordeting via `rustfmt.toml`:

https://github.com/rust-analyzer/rust-analyzer/blob/4e8664d9d37021e85d72b6228fa45e7edf4a1a74/rustfmt.toml#L1

why do we have this changes?

Reply:
Not sure. I didn't change the `rustfmt.toml` file. Maybe that's `import_layouts` or `merge_imports`?